lib/spack/spack/cmd/install.py

```diff
@@ -41,6 +41,8 @@
 from spack.package import PackageBase
 
 description = "build and install packages"
+section = "basic"
+level = "short"
```

Comment:
Ok I actually tried this.  I changed basic to `query packages` and it seems to work -- update coming.